testing/cloudi/APKBUILD

```diff
@@ -28,9 +28,9 @@
 #   (shown below with the assignment of CLOUDI_PID_FILE).
 
 pkgname=cloudi
-pkgver=1.7.2_rc1
-_srcver=70addfda7133ac209157d48e38be70be7597ce8a
-pkgrel=2
+pkgver=1.7.2_rc2
+_srcver=cb1287573a96c5e1359691da95123fbc29d32554
```

Comment:
they dont create tags for release candidates?

Reply:
I updated the pull request to use a version tag instead of the commit hash, so this is done.